cime_config/machines/config_batch.xml

```diff
@@ -514,6 +514,16 @@
 	</queues>
     </batch_system>
 
+   <batch_system MACH="chicoma-cpu" type="slurm">
+    <directives>
+     <directive>--partition=standard </directive>
+     <directive>--qos=standard </directive>
+    </directives>
+    <queues>
+      <queue walltimemax="00:30:00" nodemax="560" default="true">standard</queue>
```

Comment:
Similarly, this needs to be:
queue walltimemax="00:30:00" nodemax="96" default="true">gpu</queue>

Reply:
I think `nodemax` here is right for the CPU side.

Reply:
Same with this - we should keep the 560 and standard queue with a different project name